src/main/java/teammates/ui/template/FeedbackResultsTable.java

```diff
@@ -24,7 +26,45 @@ public FeedbackResultsTable(int fbIndex, String studentName, FeedbackSessionResu
             for (Map.Entry<String, List<FeedbackResponseAttributes>> entry : received.entrySet()) {
                 giverIndex++;
                 this.receivedResponses.add(new FeedbackResponsePersonRow(fbIndex, giverIndex, entry.getKey(), "giver",
-                                                                         entry.getValue(), result));
+                        entry.getValue(), result));
```

Comment:
Revert this whitespace-only change if possible.

Reply:
will do.